File: app/src/main/java/com/egecius/rxjava2_demo_2/MainActivity.java
package com.egecius.rxjava2_demo_2;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;


public class MainActivity extends AppCompatActivity {

    private final MyRxDelegate mMyRxDelegate = new MyRxDelegate();

    @Override
	protected void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		setContentView(R.layout.activity_main);

//        emitErrorWithoutOnErrorHandling();
//        mMyRxDelegate.emitStreamWithFailure();

        mMyRxDelegate.applyMultipleObservOn();
    }

}

File: app/src/main/java/com/egecius/rxjava2_demo_2/MainActivity.kt
package com.egecius.rxjava2_demo_2

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity


class MainActivity : AppCompatActivity() {

    private val mMyRxDelegate = MyRxDelegate()

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        //        emitErrorWithoutOnErrorHandling();
        //        mMyRxDelegate.emitStreamWithFailure();

        mMyRxDelegate.applyMultipleObservOn()
    }

}
